bluebutton-server-app/src/main/java/gov/hhs/cms/bluebutton/server/app/stu3/providers/ExplanationOfBenefitResourceProvider.java

```diff
@@ -211,12 +206,11 @@ public Bundle findByPatient(
 		PagingArguments pagingArgs = new PagingArguments(requestDetails);
 		if (pagingArgs.isPagingRequested()) {
 			/*
-			 * A page size of 0 is odd enough that we should throw an exception.
+			 * FIXME: Due to a bug in HAPI-FHIR described here
+			 * https://github.com/jamesagnew/hapi-fhir/issues/1074 paging for count=0 is not
+			 * working correctly. Review bluebutton-data-server PR #129 for necessary code
+			 * changes when this issue is resolved.
 			 */
-			if (pagingArgs.getPageSize() == 0) {
-				throw new InvalidRequestException("Invalid request - the page size should not be zero.");
-			}
-
 			int numToReturn = Math.min(pagingArgs.getPageSize(), eobs.size());
```

Comment:
```suggestion
			int endIndex = Math.min(pagingArgs.getStartIndex() + pagingArgs.getPageSize(), eobs.size());
```

This won't apply cleanly: you'll have to update the next line as well to accommodate it. Apologies.

My real concern, though, is that it _looks_ like this code was buggy but none of the tests caught it. I think Step 1 is to write a test case that proves/disproves the bug.